dart/test/event_processor/enricher/io_enricher_test.dart

```diff
@@ -25,6 +27,8 @@ void main() {
           .firstWhere((element) => element.name == 'Dart');
       expect(dartRuntime?.name, 'Dart');
       expect(dartRuntime?.rawDescription, isNotNull);
+      expect(dartRuntime!.version.toString() != Platform.version, true);
+      expect(Platform.version.contains(dartRuntime.version.toString()), true);
```

Comment:
Matchers provide better feedback on failures:
```suggestion
      expect(dartRuntime!.version.toString(), isNot(Platform.version));
      expect(Platform.version, contains(dartRuntime.version.toString()));
```